src/Hedgehog/Gen.fs

```diff
@@ -162,30 +162,15 @@ module Gen =
     //
 
     /// Generates a random number in the given inclusive range.
-    let inline range (lo : ^a) (hi : ^a) : Gen<'a> =
-        create (Shrink.towards lo) (Random.range lo hi)
+    let inline integral (range : Range<'a>) : Gen<'a> =
+        create (Shrink.towards <| Range.origin range) (Random.integral range)
 
```

Comment:
The reason I called this `integral` in Haskell is because it's named after the `Integral` type class. The general idea with the combinators in `Gen` (that I took in Haskell, and would like to apply here) is that they should be named exactly after types. F# ad-hoc polymorphism does work a bit differently though, so perhaps `integral` is OK here.